web/content/index.adoc

```diff
@@ -6,21 +6,18 @@ title: Home
 
 The following releases are supported, meaning they receive (security) updates. Visit the https://community.theforeman.org/c/support/10[support forum] for questions.
 
+* link:/release/3.9/[Foreman 3.9 & Katello 4.11]
 * link:/release/3.8/[Foreman 3.8 & Katello 4.10]
-* link:/release/3.7/[Foreman 3.7 & Katello 4.9]
 
 === Unsupported releases
 
 The future version is built in nightly.
 
 * link:/release/nightly/[Nightly]
 
-There is a release candidate available for testing.
```

Comment:
You could also prefix this line with `// ` to keep it around as an invisible comment for the next RC.

Reply:
I added the comment.